api/mapping/views.py

```diff
@@ -50,28 +44,20 @@
     DocumentFile,
     NLPModel,
     OmopField,
-    OmopTable,
     ScanReport,
     ScanReportAssertion,
     ScanReportField,
     ScanReportTable,
     ScanReportValue,
     StructuralMappingRule,
 )
-from .services import process_scan_report
-from .services_nlp import get_json_from_nlpmodel
 from .services_datadictionary import merge_external_dictionary
+from .services_nlp import get_json_from_nlpmodel
 from .tasks import (
     nlp_single_string_task,
     process_scan_report_task,
 )
 
-# to refresh/resync with loading from the database, switch to:
-# omop_lookup = OMOPDetails(load_from_db=True)
-# this will take longer, but it will recreate the csv dump of all the
-# omop fields
-omop_lookup = OMOPDetails()
```

Comment:
Wouldn't this cause the mapping rule generation to break? Since we're still using it in def generate? In this line rules_set = omop_lookup.get_rules(concepts)

Reply:
Thanks @vpnu, I have reintroduced it inside the class based view.